spec/dummy/config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :admin_users, ExceptionHunter::Devise.config
```

Comment:
Nicee. We need to find out how this works if the user already has devise installed, like with activeadmin